app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -2,20 +2,103 @@
   'use strict';
 
   angular.module('dimApp')
-    .factory('dimSearchService', function() {
+    .factory('dimSearchService', function(dimSettingsService) {
```

Comment:
@bhollis was the intention here to rid of this service? i can't remember what it was before now.